ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageCveSingle.test.js

```diff
@@ -0,0 +1,173 @@
+import withAuth from '../../../helpers/basicAuth';
+import { visitWorkloadCveOverview } from './WorkloadCves.helpers';
+import { selectors } from './WorkloadCves.selectors';
+
+describe('Image CVE Single page', () => {
+    withAuth();
+
+    it('should correctly handle ImageCVE single page specific behavior', () => {
+        // Apply global default filters
+        visitWorkloadCveOverview();
+
+        // Click any CVE link in the table to visit the ImageCVE Page.
+        cy.get('tbody tr td a:contains("CVE")').first().click();
```

Comment:
Does `td` element have `data-label` attribute from `dataLabel` prop?

Reply:
Good catch, I added those in this PR partially for ease of selecting elements. Will update.